ansible/playbooks/k8s-0-packages.yml

```diff
@@ -26,12 +27,32 @@
           - nut-client
           - power-profiles-daemon
 
-    - name: dnf - install kubernetes packages
+    - name: dnf - install crio and iproute packages
       dnf:
         name:
           - cri-o
           - cri-tools
           - iproute-tc
-          - kubernetes-client
-          - kubernetes-kubeadm
-          - kubernetes-node
+
+    - name: yum - add kubernetes repository
+      ansible.builtin.yum_repository:
+        name: Kubernetes
+        description: Kubernetes
+        baseurl: https://packages.cloud.google.com/yum/repos/kubernetes-el7-$basearch
```

Comment:
Yes this differs from the original docs. With the `..-el7-/$basearch` I ended up with 404 errors when running the yum package commands in the next task.